### PR TITLE
Error check for undefined access token

### PR DESCRIPTION
**Pull request by rtgoodwin**
After a lot of troubleshooting, I realized I'd made a very dumb easy typo when I updated my config for .12. Adding this as a way to catch this condition so you don't just mysteriously fail with 403, etc. 

Note, I'm not a JS expert, but this seemed to work; may not be idomatic. Figured I'd try to do the PR vs just complaining in an issue. 😄

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -32,6 +32,11 @@ class NestCamPlatform {
 
   didFinishLaunching() {
     let self = this;
+    let access_token = self.config['access_token'];
+    if ( typeof access_token == 'undefined' || access_token )
```
Comment (KhaosT) [inline index.js]:
```suggestion
     if ( typeof access_token == 'undefined' || accessToken )
```

File: index.js [hunk 2]
```diff
@@ -32,6 +32,11 @@ class NestCamPlatform {
...
+    {
+      throw new Error('access_token is not defined in the Homebridge config');
+    }
     self.nestAPI = new Nest(self.config['access_token']);
```
Comment (KhaosT) [inline index.js]:
```suggestion
    self.nestAPI = new Nest(accessToken);
```

**Comment by rtgoodwin** [comment]
Good edits on all; looks like I wasn't TOO far off. :) I was going to use the new var as in your last suggestion but I didn't know if I'd be committing some JS sin of reference, etc.

**Comment by KhaosT** [comment]
😅Thank you :)

**Pull request merged**